### PR TITLE
feat: introduce a svelte ui package

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
It brings a new yarn workspace with components that could be reused/shared with other projects.
It should include most of the UI components used to display and for the UI layout

Then we should publish the ui-svelte package to npmjs so other components could reuse components defined there

The renderer package has a dependency to this new module

for now, there is no component being defined but stay tuned 🎉

### Screenshot / video of UI

N/A

### What issues does this PR fix or reference?

#5757 

### How to test this PR?

should work as of today 🤞 

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: packages/ui/index.html
Comment (axel7083) [inline packages/ui/index.html:1]: What is the purpose of this HTML file in the UI packages ? Does svelte requires this ?

**Review by axel7083 — APPROVED** [review]

File: packages/ui/index.html
Comment (benoitf) [reply packages/ui/index.html:1]: I think it was vite, but as I did several attempts before issuing the PR I'll check if we really need it now that I switched to svelte-package

Comment (benoitf) [reply packages/ui/index.html:1]: removed html file

I also updated the scripts to match the reality

Comment (axel7083) [reply packages/ui/index.html:1]: Okey perfect!

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**